specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-06-01-preview/machineLearningServices.json

```diff
@@ -1270,6 +1270,57 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/computes/{computeName}/updateIdleShutdownSetting": {
+      "post": {
```

Comment:
>post [](http://example.com/codeflow?start=7&length=4)

ARM recommended approach for this is to add the idleShutdownTime as a property on ComputeInstance, so it is updated via PUT or PATCH. This doesn't seem to be a good reason to add a POST API.

Reply:
I will review this with my PM and come back to you. I added another comment to this PR and tagged you. I believe that that should be the extent of the changes here regarding having compute instance support this property on PUT, but please correct me if I'm wrong

Reply:
Talked to the PM and we're ok with not having this API, only having the CI PUT request. Will update PR